app/views/layouts/_header.html.erb

```diff
@@ -1,8 +1,10 @@
 <nav class="">
     <ul>
         <li> <%= link_to "Store Home", store_path %> </li>
-        <% if !!@cart %>
-          <li> <%= link_to "Cart", cart_path(current_cart) %></li>
+        <% #binding.pry %>
+        <% if !!current_user && !!current_user.current_cart %>
```

Comment:
Hey @pletcher just wanted to double check you wanted to add the #binding.pry to line 4? If that's intentional, cool, but just wanted to double check . . .